djstripe/models.py

```diff
@@ -26,6 +27,7 @@
 from .signals import subscription_made, cancelled, card_changed
 from .signals import webhook_processing_error
 
+_log = logging.getLogger(__name__)
```

Comment:
You're right - might as well do it right now while we're at it. First thing is to rename ``_log`` to ``logger``